devnet-sdk/devstack/sysext/helpers.go

```diff
@@ -23,31 +25,89 @@ const (
 	FeatureInterop = "interop"
 )
 
-func (orch *Orchestrator) rpcClient(t devtest.T, endpoint string) client.RPC {
+func (orch *Orchestrator) rpcClient(t devtest.T, service *descriptors.Service, protocol string) client.RPC {
+	t.Helper()
+
+	endpoint, header, err := orch.findProtocolService(service, protocol)
+	t.Require().NoError(err)
+
```

Comment:
I guess we're not too concerned about breaking the existing API right? Otherwise I'd keep the original method signature and make it wrap this method with some defaults provided

Reply:
yeah, compatibility is not a concern at all right now